styleguide/isort.toml

```diff
@@ -9,15 +9,13 @@
 
 # This file is included into base-preset.
 
-["setup.cfg".isort]
-    # These values should be placed inside `setup.cfg` in your repo:
+["pyproject.toml".isort]
+    # These values should be placed inside `pyproject.toml` in your repo:
     include_trailing_comma = true
     known_local_folder = "paradox*"
-    known_third_party = "MySQLdb,croniter,dateutil,dicttoxml,django,furl,html2text,jsonschema,lxml,markdown,phonenumbers,pytz,urlextract"
```

Comment:
I think that isort knows what package is third party https://pycqa.github.io/isort/docs/configuration/options.html#sections